packages/nextra/styles/cards.css

```diff
@@ -11,7 +11,7 @@
   }
 
   & > span > svg {
-    @apply x:w-6 x:h-auto x:transition x:text-black/30 x:dark:text-white/40 x:shrink-0;
+    @apply x:w-6 x:h-auto x:transition x:text-black/50 x:dark:text-white/50 x:shrink-0;
```

Comment:
```suggestion
    @apply x:w-6 x:h-auto x:transition x:text-black/40 x:dark:text-white/50 x:shrink-0;
```

too much on light mode

Reply:
40% fails wcag, this is about accessibility, what you think looks clear may not look clear to someone else. this is why accessibility is important and is required by law in many places

![image](https://github.com/user-attachments/assets/1ab8bfb1-0d90-41fe-8ede-576b5c70dcc0)

Reply:
Ok, let's stay with it!